lib/sensu/server/process.rb

```diff
@@ -1207,30 +1216,42 @@ def request_leader_election
         end
       end
 
-      # Set up the leader monitor. A one-time timer is used to run
-      # `request_leader_exection()` in 2 seconds. A periodic timer is
-      # used to update the leader lock timestamp if the current
-      # process is the leader, or to run `request_leader_election(),
-      # every 10 seconds. The timers are stored in the timers hash
-      # under `:run`.
-      def setup_leader_monitor
+      # Request Sensu server task elections. This method shuffles the
+      # tasks and introduces some randomized splay timers.
+      def request_task_elections
+        TASKS.shuffle.each do |task|
+          @timers[:run] << EM::Timer.new(rand * 10) do
```

Comment:
Need to replace this with something "intelligent" or serialize task election requests and hope for the best. If there are 3 Sensu servers, it would be great to get a single task on each one.